app/views/catalog/_form_basic_info.html.haml

```diff
@@ -52,7 +52,20 @@
                         :class                => "selectpicker")
           :javascript
             miqSelectPickerEvent('dialog_id', '#{url}')
-
+    - if @edit[:new][:st_prov_type] == "generic"
+      .form-group
+        %label.col-md-2.control-label
+          = _('Subtype')
+        .col-md-4
+          %p.form-control-static
+            - available_subtypes = @generic_subtypes.invert.to_a.sort_by { |a| a.first.downcase }
+            - options = [["<#{_('Choose')}>", nil]] + available_subtypes
+            = select_tag('generic_subtype',
+                          options_for_select(options, @edit[:new][:generic_subtype]),
```

Comment:
@ZitaNemeckova can you sort the values in drop down. Also should we make "custom" as a default selection and then remove <Choose> from the drop down. This was if they user doesn't make selection, Catalog Item will be saved as "Custom" subtype